composer.json

```diff
@@ -34,17 +34,14 @@
         "matthiasnoback/symfony-config-test": "^6.1",
         "matthiasnoback/symfony-dependency-injection-test": "^6.1",
         "phpstan/extension-installer": "^1.0",
-        "phpstan/phpstan": "^1.1 || ^2.0",
-        "phpstan/phpstan-phpunit": "^1.0 || ^2.0",
         "phpstan/phpstan-strict-rules": "^1.0 || ^2.0",
         "phpunit/phpunit": "^10.5.54 || ^11.5.38 || ^12.3.10",
         "psalm/plugin-phpunit": "^0.18 || ^0.19",
```

Comment:
```suggestion
```

this one is probably not needed as psalm is being phased out

Reply:
indeed :man_facepalming: did too many changes yesterday and overlooked that. Will fix it :+1: